StandardLibrary/test/TestString.dfy

```diff
@@ -47,4 +47,12 @@ module TestStrings {
     expect y == [1,2,3,4,5];
   }
 
+  // ensure that FileIO error are returned properly, and not a panic! or the like
+  // This fails to fail on Windows+Dotnet, becasue \ instead of /
+  // method {:test} TestBadFileIO()
+  // {
+  //   var x := WriteBytesToFile("/../../MyFile", [1,2,3,4,5]);
```

Comment:
Why commented code? Shouldn't we remove it?

Reply:
1) we really want this test, and I want to leave a reminder that we can't do this.
2) it's convenient to test locally by uncommenting, even if you can't run it in CI.

Reply:
We should add a comment `Enable this test if you want to test X locally`?